src/events/alb/lambda-events/LambdaALBRequestEvent.js

```diff
@@ -0,0 +1,27 @@
+export default class LambdaALBRequestEvent {
+  #routeKey = null
+  #request = null
+
+  constructor(request) {
+    this.#request = request
+  }
+
+  create() {
+    const { method } = this.#request
+    const httpMethod = method.toUpperCase()
+    return {
+      requestContext: {
+        elb: {
+          targetGroupArn:
+            'arn:aws:elasticloadbalancing:us-east-1:550213415212:targetgroup/5811b5d6aff964cd50efa8596604c4e0/b49d49c443aa999f',
```

Comment:
It seems to me that this hardcoded value needs to be removed

Reply:
definitely, let's replace it with a dummy arn for target group